docs/content/docs/ops/production_ready.md

```diff
@@ -81,4 +81,8 @@ It is a single point of failure within the cluster, and if it crashes, no new jo
 
 Configuring [High Availability]({{< ref "docs/deployment/ha/overview" >}}), in conjunction with Apache Zookeeper or Flinks Kubernetes based service, allows for a swift recovery and is highly recommended for production setups.
 
-{{< top >}}
\ No newline at end of file
+### Secure Flink Cluster Access
+
+To prevent potential security vulnerabilities, such as arbitrary code execution, ensure that your Flink clusters are only accessible to trusted users. Flink clusters should not be exposed to the company intranet or public internet. For more details, refer to the [Flink Security FAQ](https://flink.apache.org/what-is-flink/security/#during-a-security-analysis-of-flink-i-noticed-that-flink-allows-for-remote-code-execution-is-this-an-issue).
```

Comment:
it seems very restrictive to say `Flink clusters should not be exposed to the company intranet`. If they are locked down with TLS and appropriate authentication - this could be acceptable, rather than having to have its own network to run Flink.  We could have a statement around TLS and better still mTLS for all connections.

Reply:
Thank you @davidradl. 

i have incorporated your suggestion. Please review whenever time